spec/commands/hist_spec.rb

```diff
@@ -132,6 +132,16 @@ def next_input
     expect(out).to match(/b\n\d+:.*c\n\d+:.*d/)
   end
 
+  it 'should show lines between offsets A and B with the --show switch' do
+    ("a".."z").each do |v|
+      @hist.push v
+    end
+
+    out = @t.eval 'hist --show -4..-1'
+    expect(out.each_line.count).to eq 4
+    expect(out).to match(/w\n\d+:.*x\n\d+:.*y\n\d+:.*z/)
```

Comment:
Couple comments:

* there's no need to build the whole alphabet from A to Z. The A to, say F, range is more than enough
* the regexp here could be replaced with a plain string, which is IMO a bit more readable

Reply:
@kyrylo 👍, updated. The `"a".."z"` range and regexp were to be consistent with the style of the previous two tests, but I see now those comments explain the first line situation, not applicable here.